04week/tomorrows-technology/css/style.css

```diff
@@ -0,0 +1,157 @@
+body{
+  /* margin-top: 25px; */
+  width: 100%;
+  height: 100%;
+  /* padding-left: 20px;
+  padding-right: 20px;
+  padding-top: 20px; */
+  display: flex;
+  font-family: sans-serif;
+}
+
+#img-container{
+  width: 65%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+#content-container{
+  /* padding-left: 50px; */
+  width: 25%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+h1{
+  font-size: 35px;
+  display: flex;
+  justify-content: flex-start;
+}
+
+h2{
+  font-weight: bold;
+  margin-top: 0;
+}
+
+h3{
+  font-size: 20px;
+  margin-top: 0;
+}
+
+p{
+  font-size: 15px;
+  font-weight: bold;
+}
+
+#border-p{
+  border-top: 1px solid black;
+  padding-top: 20px;
+  margin-top: 0;
+}
+
+#button1{
+  display: flex;
+  width: 120px;
+  height: 40px;
+  text-align: center;
+  font-weight: bold;
+  color: white;
+  background-color: gray;
+  border: none;
+  font-size: 15px;
+}
+
+footer{
+  visibility: hidden;
+}
+
+@media (max-width: 768px) {
+  body{
```

Comment:
try to think of another way to change the view.. adjusting the body between queries creates a weird experience when resizing